admin/server/organizations.go

```diff
@@ -427,6 +427,8 @@ func (s *Server) AddOrganizationMemberUser(ctx context.Context, req *adminv1.Add
 		return nil, status.Error(codes.PermissionDenied, "as a non-admin you are not allowed to assign an admin role")
 	}
 
+	// TODO if the role is non guest then check if the user has any resources level access on projects in the org directly or through usergroup and clear those
```

Comment:
Why would it do that?

To be mindful of safety, I don't think it should ever clear project-level resource access when changing org-level roles. Only changing project-level roles should clear resource-level access.

Reply:
ok